passes/techmap/muxcover.cc

```diff
@@ -631,7 +631,7 @@ struct MuxcoverPass : public Pass {
 		log("\n");
 		log("Cover trees of $_MUX_ cells with $_MUX{4,8,16}_ cells\n");
```

Comment:
Shouldn't this be `$_MUX{2,4,8,16}_`?

Reply:
Good spot! `$_MUX_` cells are actually 2:1 multiplexers. I'm adding the ability to change its cost base (rather than be hardcoded to 100) for use in decisions about covering with the 4/8/16:1 cells.

Reply:
That makes sense, but I thought the documentation was a little confusing here.

Reply:
Fair point! I've updated the doc -- is it clearer now?

Reply:
Yes it is, thank you!